grunt/tasks/create-json-config.js

```diff
@@ -1,39 +1,49 @@
 /**
 * This is a simple function to take the course's config.json
-* and append the theme and menu .json
+* and append the config.json (or legacy theme.json and menu.json) from plugins folders
 */
+var path = require("path");
+
 module.exports = function(grunt) {
+    
     grunt.registerTask('create-json-config', 'Creating config.json', function() {
-        var customItems = ['theme', 'menu'];
-        var configJson = grunt.file.readJSON(grunt.config('sourcedir') + 'course/config.json');
 
-        customItems.forEach(function (customItem) {
-            // As any theme folder may be used, we need to first find the location of the
-            // theme.json file
+        var sourcedir = grunt.config('sourcedir');
```

Comment:
Please can you update as per https://github.com/adaptlearning/adapt_framework/pull/1779#discussion_r142157671?